nada_numpy/array.py

```diff
@@ -870,28 +870,6 @@ def item(self, *args, **kwargs):
             return NadaArray(result)
         return result
 
-    @overload
```

Comment:
@jcabrero why are we deleting this?

Reply:
In version 2.0.0 of Numpy `itemset` is deprecated. They suggest using `array[item] = value` instead of `itemset`. If there is a use we need to keep, maybe we can find another way of keeping this behaviour.

Reply:
nice, sounds good 👍